day3/src/reducers.js

```diff
@@ -47,7 +47,7 @@ const cardReducer = (state = initialStateCards, action) => {
 };
 
 const initialStateGame = {
-  finished: false,
+  finished: true,
```

Comment:
Oups, jeg satt bare den for å teste. Orket ikke å spille hele spillet :P Bra catch! :-D